src/main/host/memory_manager/memory_copier.rs

```diff
@@ -178,8 +176,8 @@ impl MemoryCopier {
 
         let towrite = src.len();
         trace!("write_ptr writing {} bytes", towrite);
-        let local = [nix::sys::uio::IoVec::from_slice(src)];
-        let remote = [nix::sys::uio::RemoteIoVec {
+        let local: [std::io::IoSlice; 1] = [std::io::IoSlice::new(src)];
+        let remote: [nix::sys::uio::RemoteIoVec; 1] = [nix::sys::uio::RemoteIoVec {
             base: usize::from(dst.ptr()),
             len: towrite,
```

Comment:
Why are the types specified here? (I don't mind having the type specified, I'm just wondering if there's a reason for it.)

Reply:
It's probably overkill; removed.

I have VSC set up to show me the types in places like this, but leaving them implicit sometimes makes the code less readable outside of VSC. I think I experimented with the shortcut to make the types explicit while I was editing this, but it's not really needed in this case since they're already explicit on the RHS